helpers/documentation_helper.js

```diff
@@ -60,14 +60,14 @@ function parseCommentNodes(commentNodes) {
     // Only process documentation blocks
     .filter(commentNode => commentNode.value.match(/^\*[\s\S]*@component/))
     // Extract @annotations
-    .map(commentNode => commentNode.value.match(/@[^@]+/g))
+    .map(commentNode => commentNode.value.match(/@(component|description|example|param|property|return|see)\s+[^@]+/g))
```

Comment:
Oh! I meant to call that out

I grepped the entire source code and those are the tags actually used.